apps/nextjs-website/src/components/organisms/WebinarsSection/WebinarsSection.tsx

```diff
@@ -0,0 +1,82 @@
+'use client';
+import { Webinar } from '@/lib/types/webinar';
+import { Box, Typography, useTheme } from '@mui/material';
+import React from 'react';
+import WebinarCard from '@/components/molecules/WebinarCard/WebinarCard';
+import LinkButton from '@/components/atoms/LinkButton/LinkButton';
+import EContainer from '@/editorialComponents/EContainer/EContainer';
+
+export type webinarsSectionProps = {
+  title: string;
+  description: string;
+  link: { href?: string; label: string };
```

Comment:
This field should be optional
![image](https://github.com/pagopa/developer-portal/assets/24475319/635d057b-9075-4b28-91bc-0b3e38b3fbd8)

```suggestion
  link?: { href?: string; label: string };
```